addons/web/static/src/scss/bootstrap_overridden_frontend.scss

```diff
@@ -69,6 +69,11 @@ $input-disabled-border-color: rgba($input-color, 0.15) !default;
 
 $form-check-input-border: $border-width solid $gray-400 !default;
 $form-check-input-border-radius: 0 !default;
+$form-check-input-checked-color: color-contrast($primary, $body-color, $white)!default;
```

Comment:
- Missing space to be consistent with the rest :wink: 
- Sure using `$primary` is the best here? Maybe `$component-active-bg` is better? (as it is also overridden in website?)

Reply:
Indeed, I've updated the code 👍